File: #6 - Clean Architecture Version (Getx)/lib/features/comment/domain/entities/comment_entity.dart
import 'package:flutter/foundation.dart' show immutable;

@immutable
class CommentEntity {
  const CommentEntity({
    required this.id,
    required this.postId,
    required this.name,
    required this.email,
    required this.body,
  });

  final int id;
  final int postId;
  final String name;
  final String email;
  final String body;
}

File: gorest-backend/src/main/kotlin/com/sinasys/gorest/backend/entity/CommentEntity.kt
package com.sinasys.gorest.backend.entity

import jakarta.persistence.Id
import jakarta.persistence.Table
import jakarta.persistence.Entity
import jakarta.persistence.Column
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType


@Entity
@Table(name = "comment")
open class CommentEntity(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    open var id: Long = 0,
    @Column(nullable = false)
    open var postId: Long? = 0,
    @Column(nullable = false)
    open var name: String = "",
    @Column(nullable = false)
    open var email: String = "",
    @Column(nullable = false)
    open var body: String = ""
)
